editor/editor_plugin_settings.cpp

```diff
@@ -62,44 +63,33 @@ void EditorPluginSettings::update_plugins() {
 	plugins.sort();
 
 	for (int i = 0; i < plugins.size(); i++) {
-		Ref<ConfigFile> cf;
-		cf.instantiate();
+		Ref<ConfigFile> cfg;
+		cfg.instantiate();
 		const String &path = plugins[i];
 
-		Error err2 = cf->load(path);
+		Error err = cfg->load(path);
 
-		if (err2 != OK) {
-			WARN_PRINT("Can't load plugin config: " + path);
+		if (err != OK) {
+			WARN_PRINT("Can't load plugin config at " + path);
```

Comment:
```suggestion
			WARN_PRINT("Can't load plugin config at: " + path);
```
I'd keep the colon for clarity (we use it generally)